### DISCUSSION TITLE
How to download BridgDB & how to implement reversible reactions?

### DISCUSSION THREAD

**Discussion by JuliaBatUM** [wikipathways/wikipathways-help#66] [Q&A] [answered]
Hey!
I have again some questions about PathVisio :') 
I tried downloading BridgeDB according to the guidelines on Pathvisio : https://pathvisio.org/pathvisio-book/load-identifier-mapping-database.html?q=bridg#load-identifier-mapping-database

I downloaded the “homo sapiens” file: https://data.bridgedb.org/gene_database/

Unfortunately, I cannot open the file on Pathvisio the same way it is shown in the pathvisio-guideline-video, because I don’t have the **Load identifier Mapping** button  (see picture below). I also saw that I have version 3.3.0 but, in the video, version 4.0.0. is used. Could this be the reason?
![image](https://github.com/wikipathways/wikipathways-help/assets/130973169/3a062069-9c8a-46d3-b005-5406fa9281a5)
I checked for version 4.0.0, and it seems that it is still under construction? I also tried to open the file by clicking on the file and then selecting the app in which it should open, and then I got this error message in Pathvisio:
![image](https://github.com/wikipathways/wikipathways-help/assets/130973169/e0bc6d59-c65c-4e82-a8bf-a6b830037468)
 Is there a way to use BridgeDB?

And to my second question, is there a way to represent reversible reactions (in two directions)?

Thank you very much for your help,
Julia

**Comment by mkutmon** [comment]
I apologies for the confusion. The PathVisio Book is only for the not-yet-released PathVisio 4 version. We are working hard on getting that ready - I just saw that the Book is mentioned on the Documentation page of PathVisio - I will remove it from there for now. 

In PathVisio 3.3.0, you load a mapping database, by clicking Data > Import Gene/Metabolite Expression Database. Please let me know if that doesn't work.

**Reply by JuliaBatUM** [reply]
Hi!
Thanks for the help.
 I only have the button **Import Expression Data**, and uploading the file like that does not work. It looks like that:
![image](https://github.com/wikipathways/wikipathways-help/assets/130973169/73f90e75-3966-4771-bdd8-f9bb9b1dbf20)
At **Input file**, I browsed through my computer to add the "homo sapiens " file :
![image](https://github.com/wikipathways/wikipathways-help/assets/130973169/0094c1fb-6f26-4677-8952-14af4d0acf3a)
I always clicked on next, but I don´t think it is supposed to look like that.

I also tried to open the file under **Gene/Metabolite/interaction database**, but I got an error message:
![image](https://github.com/wikipathways/wikipathways-help/assets/130973169/19a90f3d-81e4-4867-8736-84f0875a5d25)

**Reply by JuliaBatUM** [reply]
**Update**: I succeeded in installing BridgDB by following the guidelines from the Wikipathway Academy: https://academy.wikipathways.org/stages/walk-install-pv/

**Comment by mkutmon** [comment, marked as answer]
Regarding the reversible reactions, you have two options. 
* Create two conversion lines (best options if different enzymes are used)
* Change the start line type to a mim-conversion

![image](https://github.com/wikipathways/wikipathways-help/assets/2158343/763c2737-43dd-49f6-9198-74e490db0123)
